src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
+
+newTh.outerHTML = targetTh.outerHTML;
+targetTh.innerHTML = th.innerHTML;
+
+const tf = table.tFoot.rows[0].cells[1];
+const targetTf = table.tFoot.rows[0].cells[4];
+const newTf = table.tFoot.rows[0].insertCell(5);
+
+newTf.outerHTML = targetTf.outerHTML;
+targetTf.innerHTML = tf.innerHTML;
```

Comment:
Swapping innerHTML directly can lead to loss of event listeners or other dynamic properties. Consider whether this is the desired outcome.